ipaserver/plugins/groupfas.py

```diff
@@ -221,3 +221,4 @@ def group_show_fas_postcb(self, ldap, dn, entry_attrs, *keys, **options):
 group_show.register_post_callback(group_show_fas_postcb)
 
 i18n_messages.messages["groupfas"] = {"name": _("Fedora Account System")}
+i18n_messages.messages["fasgroup"] = {"name": _("FAS Group")}
```

Comment:
It's a bit weird to have both `groupfas` and `fasgroup`, maybe keep both strings in the latter and name the first message "fas"?